eclair-rpc/README.md

```diff
@@ -0,0 +1,27 @@
+# Bitcoin-s Eclair RPC client
```

Comment:
It is probably worth mentioning somewhere in this readme that a bitcoind instance has to be running to run the eclair jar (and probably link to the bitcoind rpc client)